examples/hello-world/hello_service.py

```diff
@@ -19,30 +19,35 @@ async def get_payload():
         )
 
     async def start(self):
+        async for s in super().start():
+            yield s
```

Comment:
Ugh

Reply:
?

Reply:
anyway, _yes_ it's an "ugh" from me too... that's _exactly_ what I complained about re: the "un-implicitization" movement ;)